horovod/spark/keras/estimator.py

```diff
@@ -160,6 +160,8 @@ class KerasEstimator(HorovodEstimator, KerasEstimatorParamsReadable,
 
     custom_objects = Param(Params._dummy(), 'custom_objects', 'custom objects')
     _keras_pkg_type = Param(Params._dummy(), '_keras_pkg_type', 'keras package type')
+    model_checkpoint_callback_fn = Param(Params._dummy(), 'model_checkpoint_callback_fn',
```

Comment:
I think `checkpoint_callback` would be a better name, to be consistent with PyTorch Lightning.